### PR TITLE
45491 Changes to support lorota security updates partial dial-up

### PR DESCRIPTION
**Pull request by kanchanasuriya**
## Description of change

Removed lorota security update feature flag check to unblock partial dial-up.  Implemented auth parameter changes based on the payload from front-end

## Original issue(s)
department-of-veterans-affairs/va.gov-team#45491

## Things to know about this PR

- [x] rubocop
- [x] rspec
- [x] E2E testing in locally running vets-website

### CODE REVIEW

**Review by rileyanderson — APPROVED** [review]

File: config/features.yml [hunk 1]
```diff
@@ -95,7 +95,7 @@ features:
     description: Enables disclaimer for possible untranslated content on spanish pages
     enable_in_development: true
   check_in_experience_lorota_security_updates_enabled:
```
Comment (randomsync) [inline config/features.yml:97]: do we still need to keep it since we're not using it in the code anymore? is it used by website?

**Review by randomsync — COMMENTED** [review]
looks good, just a minor question if we need to keep the feature flag.

File: config/features.yml (hunk 1 above)
Comment (kanchanasuriya) [reply config/features.yml:97]: Yeah, that's right.  It is used by the website for partial dial-up.

**Review by randomsync — APPROVED** [review]

**Comment by randomsync** [comment]
Only other thing I'd suggest is to change the title of the commit to more accurately reflect the changes in this PR. Thanks.

**Pull request merged**